[Bug] fe query error in local run mode

### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/incubator-doris/issues?q=is%3Aissue) and found no similar issues.


### Version

master

### What's Wrong?

2022-04-23 10:21:21,787 WARN (doris-mysql-nio-pool-0|129) [BackendServiceProxy.cancelPlanFragmentAsync():131] Cancel plan fragment catch a exception, address=127.0.0.1:8060
java.lang.NoClassDefFoundError: io/grpc/Channel
	at org.apache.doris.rpc.BackendServiceProxy.getProxy(BackendServiceProxy.java:87) ~[classes/:?]
	at org.apache.doris.rpc.BackendServiceProxy.cancelPlanFragmentAsync(BackendServiceProxy.java:128) ~[classes/:?]
	at org.apache.doris.qe.Coordinator$BackendExecState.cancelFragmentInstance(Coordinator.java:1986) ~[classes/:?]
	at org.apache.doris.qe.Coordinator.cancelRemoteFragmentsAsync(Coordinator.java:868) ~[classes/:?]
	at org.apache.doris.qe.Coordinator.cancelInternal(Coordinator.java:858) ~[classes/:?]
	at org.apache.doris.qe.Coordinator.sendFragment(Coordinator.java:640) ~[classes/:?]
	at org.apache.doris.qe.Coordinator.exec(Coordinator.java:531) ~[classes/:?]
	at org.apache.doris.qe.StmtExecutor.handleQueryStmt(StmtExecutor.java:976) ~[classes/:?]
	at org.apache.doris.qe.StmtExecutor.execute(StmtExecutor.java:390) ~[classes/:?]
	at org.apache.doris.qe.StmtExecutor.execute(StmtExecutor.java:307) ~[classes/:?]
	at org.apache.doris.qe.ConnectProcessor.handleQuery(ConnectProcessor.java:212) ~[classes/:?]
	at org.apache.doris.qe.ConnectProcessor.dispatch(ConnectProcessor.java:349) ~[classes/:?]
	at org.apache.doris.qe.ConnectProcessor.processOnce(ConnectProcessor.java:538) ~[classes/:?]
	at org.apache.doris.mysql.nio.ReadListener.lambda$handleEvent$0(ReadListener.java:50) ~[classes/:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) ~[?:1.8.0_242]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) ~[?:1.8.0_242]
	at java.lang.Thread.run(Thread.java:748) ~[?:1.8.0_242]
Caused by: java.lang.ClassNotFoundException: io.grpc.Channel
	at java.net.URLClassLoader.findClass(URLClassLoader.java:382) ~[?:1.8.0_242]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:419) ~[?:1.8.0_242]
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:352) ~[?:1.8.0_242]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:352) ~[?:1.8.0_242]
	... 17 more

### What You Expected?

query success

### How to Reproduce?

use master branch, cd fe && mvn clean install -DskipTests  -Dcheckstyle.skip=true, run fe 

### Anything Else?

_No response_

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

--- a/fe/pom.xml
+++ b/fe/pom.xml
@@ -459,19 +459,16 @@ under the License.
                 <groupId>io.grpc</groupId>
                 <artifactId>grpc-netty-shaded</artifactId>
                 <version>${grpc.version}</version>
-                <scope>provided</scope>
             </dependency>
             <dependency>
                 <groupId>io.grpc</groupId>
                 <artifactId>grpc-protobuf</artifactId>
                 <version>${grpc.version}</version>
-                <scope>provided</scope>
             </dependency>
             <dependency>
                 <groupId>io.grpc</groupId>
                 <artifactId>grpc-stub</artifactId>
                 <version>${grpc.version}</version>
-                <scope>provided</scope>
             </dependency>
             <!-- https://mvnrepository.com/artifact/com.google.protobuf/protobuf-java -->
             <dependency>

